devops/ansible/ssl.yml

```diff
@@ -0,0 +1,39 @@
+- name: Install certbot
+  apt:
+    name: certbot
+    update_cache: yes
+  become: true
+
+- name: Copy post cert renewal hook script
+  copy:
+    src: arango-ssl.sh
+    dest: /etc/letsencrypt/renewal-hooks/deploy/
+    mode: u=rwx,g=,o=
+  become: true
+
+- name: Set server name inside of the post cert renewal hook script
+  replace:
+    path: /etc/letsencrypt/renewal-hooks/deploy/arango-ssl.sh
+    regexp: '(SERVER_NAME)'
+    replace: '{{ server_name }}'
+  become: true
+
+- name: Generate a certificate
+  shell:
+    cmd: certbot certonly -d {{ server_name }} --standalone -m jwilburn@sci.utah.edu --agree-tos -n
```

Comment:
It seems like your email address should be an input value (either via prompt or environment variable or something).

Reply:
Fixed in 0dbcfbd